modules/update-oc-configmap-signature-verification.adoc

```diff
@@ -18,7 +18,12 @@ If you are upgrading from a release prior to version 4.4.8, you must use the man
 
 .Procedure
 
-. Obtain the image signature for the version that you are upgrading to from either link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release[mirror.openshift.com] or link:https://storage.googleapis.com/openshift-release/official/signatures[Google Cloud Storage (GCS)].
+. Obtain the image signature for the version that you are upgrading to:
++
+[source,terminal]
+----
+$ oc adm release mirror --release-image-signature-to-dir 
```

Comment:
`oc adm release mirror --release-image-signature-to-dir` lacks needed arguments, please see `oc adm release mirror -h` for the flag and example usage. Reassigning to OTA subteam Jia Liu for further review and the follow-up updates
/assign @jiajliu